### DISCUSSION TITLE
Lumix V-Log to GoPro MLOG

### DISCUSSION THREAD

**Discussion by Jeje2607** [gopro/labs#477] [Q&A]
Hello everyone !

I would like to use Gopro Labs LOG function for GoPro 11 with V-Log emulation (Lumix Varicam)
Does anyone have any values ​​for me?
Thank you so much

**Comment by dnewman-gpsw** [comment]
I would recommend that, just normalize cameras in their optimal log encoding with LUTs after capture.  But it want to do this you will have to determine the math for V-Log then add that equation here https://www.desmos.com/calculator/sfxqtunzty  Then move the sliders to match V-Log.

**Comment by Jeje2607** [comment]
Thank you very much for your answer. Do you know where I can find the equation?
